graphql/schema.graphql

```diff
@@ -1014,7 +1016,9 @@ type BaseTaskInfo {
 }
 
 type GroupedProjects {
-  name: String!
+  repoDisplayName: String! 
```

Comment:
Can we call this groupDisplayName instead? Otherwise it's confusing why this isn't just repo.DisplayName

Reply:
I like that. done.